sdk/test-utils/recorder-new/MIGRATION.md

```diff
@@ -0,0 +1,160 @@
+# Migration Guide
+
+This document outlines key differences between the legacy recorder and the new Unified Recorder client.
+
+## Installing the Unified Recorder
+
+The first step is to install the unified recorder client library in your package by running
+
+```bash
+$ rush add --dev -p @azure-tools/test-recorder-new
+```
+
+You will use this library to control the recorder from your tests. The API is somewhat similar to the legacy recorder, albeit with some important differences. These will be discussed below.
+
+**⚠️ Important:** Do not uninstall the old recorder (`@azure-tools/test-recorder`) just yet! At this time, there are still utilities from the legacy recorder package that have not been migrated to the new package.
+
+## Changes to NPM scripts
+
+For the unified recorder client library to work, the [test proxy server] must be active while you are running your tests. Helpers have been added to the `dev-tool` package which manage starting and stopping the test proxy server before and after your tests are run. These helpers use the test proxy server Docker image. Therefore, Docker must be installed in your development environment.
+
+The script defined in `package.json` to run the browser tests should be replaced with `dev-tool run test:browser`, and the script to run the Node tests should look something like `dev tool run test:node-ts-input --mocha=\"--timeout 1200000 'test/**/*.spec.ts'`. In the case of the Node tests, the `--mocha="..."` parameter can be used to specify arguments to be passed through to Mocha. For example:
```

Comment:
This paragraph seems redundant given the table below. Should it focus instead on introducing the table below? It could provide some perspective on how to read the table.

(The table is very readable, but this paragraph could be more useful to the table below anyway)